corehq/apps/userreports/reports/specs.py

```diff
@@ -396,7 +397,7 @@ def inner(*args, **kwargs):
         def _raw(data):
             if data['denom']:
                 try:
-                    return round(float(data['num']) / float(data['denom']), 3)
+                    return round(data['num'] / data['denom'], 3)
```

Comment:
looks related to http://python3porting.com/differences.html#rounding-behavior

Reply:
To guarantee the same behavior between python 2 and 3, we could wrap this expression in a ```float()```.  Probably not a huge deal though.

Reply:
> In Python 3 rounding of halfway cases are now always towards the nearest even. This is standard practice, as it will make a set of evenly distributed roundings average out.

Fascinating. I wonder if this is how second-graders learn to round nowadays.

I'm on board with being conservative. Will wrap in a `float()`.